manim/_config/utils.py

```diff
@@ -544,6 +545,7 @@ def digest_parser(self, parser: configparser.ConfigParser) -> "ManimConfig":
             "movie_file_extension",
             "background_color",
             "webgl_renderer_path",
+            "jupyter_width",
```

Comment:
Here the ManimConfig parser object is setting attributes specific to "CLI" as you can see on line 550 with `setattr(....)`, this should be done elsewhere. See how it's done for ffmpeg_loglevel on [line 575](https://github.com/ManimCommunity/manim/blob/6b35cb8eeca9e8af3d48615a6fec28937d2cfbb2/manim/_config/utils.py#L575)
```suggestion
```

Also see how ffmpeg_loglevel is handled in `digest_args` for an example how to set that on [line 843](https://github.com/ManimCommunity/manim/blob/6b35cb8eeca9e8af3d48615a6fec28937d2cfbb2/manim/_config/utils.py#L843)